src/frontend/components/Felleskomponenter/SkjemaFeiloppsummering/SkjemaFeiloppsummering.tsx

```diff
@@ -42,7 +47,8 @@ export const SkjemaFeiloppsummering: React.FC<Props> = ({
             (felt.id === OmDegSpørsmålId.borPåRegistrertAdresse &&
                 (felt.verdi === ESvar.NEI ||
                     (!søknad.søker.adresse && !søknad.søker.adressebeskyttelse))) ||
-            felt.id === VelgBarnSpørsmålId.velgBarn ? (
+            felt.id === VelgBarnSpørsmålId.velgBarn ||
+            (felt.id === VelgBarnSpørsmålId.leggTilBarnErFødt && felt.verdi === ESvar.NEI) ? (
             felt.feilmelding
```

Comment:
Ikke en kommentar til denne PRen, men hadde vært digg å gjort det over en smule lettere å lese ^^ trukket det ut i noen funksjoner, og kanskje "vridd det rundt". Type if true then \<SpråkTekst... else felt.feilmelding

Reply:
Så veldig enig! Men jeg lurer på om kanskje hele denne faktisk kan tas bort når vi begynner å se på feilmeldinger, fordi nå rendrer den spørsmål i feilboksen, som i "du må rette opp i: **spørsmål"**. Men tror det skal byttes ut med at vi faktisk renderer spesifikke feilmeldinger for hvert felt